.github/workflows/group-3-vpc.code-workspace

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Could you remove this file please, I should not be pushed to git.

Reply:
Removed the file.